MediaPlayer/MediaControls.module.less

```diff
@@ -55,6 +55,12 @@
 				}
 			}
 		}
+
+		&.loop {
+			.bg {
+				background-color: @agate-accent;
```

Comment:
Which GUI guide page should I refer to verify this change?

Reply:
The GUI does not have any page where the repeat functionality is activated. We chosed the bg-color.

Reply:
Maybe we can ask for clarification from GUI team and add the provided color in the PR of FORTECH-97, which is under development.

Reply:
Sure thanks!